CondFormats/HGCalObjects/interface/HGCalMappingCellIndexerTrigger.h

```diff
@@ -0,0 +1,181 @@
+#ifndef CondFormats_HGCalObjects_interface_HGCalMappingCellTriggerParameterIndex_h
+#define CondFormats_HGCalObjects_interface_HGCalMappingCellTriggerParameterIndex_h
```

Comment:
The include guard should better reflect the name of the file
```suggestion
#ifndef CondFormats_HGCalObjects_interface_HGCalMappingCellIndexerTrigger_h
#define CondFormats_HGCalObjects_interface_HGCalMappingCellIndexerTrigger_h
```